src/components/chat-item/chat-item-tree-file.ts

```diff
@@ -96,4 +114,39 @@ export class ChatItemTreeFile {
       ]
     });
   }
+
+  private readonly showTooltip = (content: string, vDir?: OverlayVerticalDirection, hDir?: OverlayHorizontalDirection): void => {
+    if (content.trim() !== undefined) {
+      clearTimeout(this.fileTooltipTimeout);
+      this.fileTooltipTimeout = setTimeout(() => {
+        this.fileTooltip = new Overlay({
+          background: true,
+          closeOnOutsideClick: false,
+          referenceElement: this.render,
+          dimOutside: false,
+          removeOtherOverlays: true,
+          verticalDirection: vDir ?? OverlayVerticalDirection.TO_TOP,
+          horizontalDirection: hDir ?? OverlayHorizontalDirection.CENTER,
+          children: [
+            new Card({
+              border: false,
+              children: [
+                new CardBody({
+                  body: content
+                }).render
+              ]
+            }).render
+          ],
+        });
+      }, PREVIEW_DELAY);
+    }
+  };
+
+  public readonly hideTooltip = (): void => {
+    clearTimeout(this.fileTooltipTimeout);
+    if (this.fileTooltip !== null) {
```

Comment:
Is this `if` necessary? The conditional code paths only differ between `null` and `undefined` now, but the effect of both is effectively the same.

Reply:
We're manually setting it to null, but would be better to make it as `this.fileTooltip != null`. Thanks for catching it up.